Kernel/FourVelocity.m

```diff
@@ -9,55 +9,86 @@
 
 
 BeginPackage["KerrGeodesics`FourVelocity`",
-	{"KerrGeodesics`ConstantsOfMotion`"}];
+	{"KerrGeodesics`ConstantsOfMotion`",
+	"KerrGeodesics`OrbitalFrequencies`"}];
 
 KerrGeoFourVelocity::usage = "KerrGeoVelocity[a,p,e,x] returns the four-velocity components as parametrized functions.";
 
 Begin["`Private`"];
 
 
-(* ::Subsection::Closed:: *)
+(* ::Subsection:: *)
 (*Error messages*)
 
 
 KerrGeoFourVelocity::parametrization = "Parameterization error: `1`"
 
 
-(* ::Section::Closed:: *)
-(*Kerr*)
+(* ::Section:: *)
+(*Schwarzschild*)
 
 
-(* ::Subsection:: *)
-(*Generic (Mino)*)
+(* ::Subsection::Closed:: *)
+(*Circular, Equatorial*)
+
+
+KerrGeoVelocityMino[(0|0.),p_,(0|0.),x_,initPhases_ ]:= Module[{En,L,Q,r,z,r1,r2,r3,r4,kr,zp,zm,kz, \[CapitalUpsilon]r, \[CapitalUpsilon]z, 
```

Comment:
The use of (0|0.) misses the case where `a` or `e` are non-machine precision zero's (e.g. 0.``32) I suggest the following instead
```mathematica
KerrGeoVelocityMino[a_,p_,e_,x_,initPhases_ ] /; a==0 && e==0
```

Reply:
I see in the rest of the package ?PossibleZeroQ is used. This should do the same thing, but I also think it's better style to be consistent throughout the package and use

```mathematica
KerrGeoVelocityMino[a_?PossibleZeroQ,p_,e_?PossibleZeroQ,x_,initPhases_ ] := ....
```

Reply:
PossibleZeroQ could be used here, although note the slightly different. PossibleZeroQ prints a message and returns True if a number might be zero but it's not sure, whereas the version with Equal will not return True in that case, which is equivalent to returning False for the purposes of the function definition. There are also circumstances where PossibleZeroQ will get it wrong (see the docs for example). These are all very much corner cases, so I think either PossibleZeroQ or Equal would be fine.